doc/source/user_guide/computation.rst

```diff
@@ -597,6 +597,18 @@ You can view other examples of ``BaseIndexer`` subclasses `here <https://github.
 
 .. versionadded:: 1.1
 
+One subclass of note within those examples is the ``NonFixedVariableWindowIndexer`` that allows
```

Comment:
we should rename this Indexer, its a bit redundant (NonFixedVariable)......followon ok